data/noStoppingSignStyle.yml

```
@@ -95,7 +96,7 @@ PE: mutcd
 PG: australia
 PH: mutcd
 PK: vienna
-PR: mutcd
+PR: mutcd text spanish # https://github.com/streetcomplete/countrymetadata/issues/1
```

```suggestion
PR: mutcd text spanish # https://www.google.de/maps/@18.4007917,-66.0561823,3a,15.4y,30.12h,80.43t/data=!3m9!1e1!3m7!1sLXFXKKxgWmlOqnv3KKWfXg!2e0!7i13312!8i6656!9m2!1b1!2i41
```

I think it is better to give the source directly.